cpp/afterburner/EicConfigurator.cc

```diff
@@ -11,6 +11,18 @@ using namespace CLHEP;
 ab::AfterburnerConfig ab::EicConfigurator::preset_ip6_eau_110x18() {
     ab::AfterburnerConfig cfg;
 
+    // Beam energy
+    cfg.hadron_beam.energy = 110;
+    cfg.lepton_beam.energy = 18;
+
+    // Beam charge
+    cfg.hadron_beam.charge = 79;
+    cfg.lepton_beam.charge = -1;
```

Comment:
Can be derived from PDG id?

Reply:
I will replace the mass and charge with just the PDG id, but inheriting from the identified kinematics would reduce the functionality of the code. At the moment you can both automatically identify the energies and particles or manually specify the configuration by name. This would limit to the automatic case which is much nicer from a coding point of view, depends what people use at the moment.

Reply:
Now just adds the PDG id of the beam particles from the first event to the run attributes